src/app/containers/ServiceWorker/index.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ServiceWorkerContainer from './index';
+
+const mockRootElement = <div />;
+document.getElementById = jest.fn().mockReturnValue(mockRootElement);
+
+describe('Service Worker', () => {
+  beforeEach(() => {
+    global.navigator.serviceWorker = {
+      register: jest.fn(),
+    };
+  });
+
+  describe('on production environment', () => {
+    beforeEach(() => {
+      process.env.NODE_ENV = 'production';
+      mount(<ServiceWorkerContainer />);
+    });
+
+    it('should be installed', async () => {
+      expect(navigator.serviceWorker.register).toHaveBeenCalled();
```

Comment:
Can you expand these tests please, so its '.toHaveBeenCalledWith(swPath)`, would be nice to test that it gets the URL from the context